tests/functional/test_plugins_config.py

```diff
@@ -10,299 +10,246 @@
-yyyy
-""",
+    "text": "new new\nxxx\n",
+    "file": "new new new\nyyyy\n",
     "string": "double new",
     "json": [{"x": "new"}, {"y": "z"}],
+    'dummy': 'donthurtme',
     "map": {"one": "two", "two": "new"},
     "list": ["one", "new", "three"],
 }
-
-INITIAL_CLUSTERS_CONFIG = {
-    'first': {
-        'config': copy.deepcopy(INITIAL_CONFIG),
-        'services': {
-            'First': copy.deepcopy(INITIAL_CONFIG),
-            'Second': copy.deepcopy(INITIAL_CONFIG),
-        },
-    },
-    'second': {
-        'config': copy.deepcopy(INITIAL_CONFIG),
-        'services': {
-            'First': copy.deepcopy(INITIAL_CONFIG),
-            'Second': copy.deepcopy(INITIAL_CONFIG),
-        },
-    },
-    'third': {
-        'config': copy.deepcopy(INITIAL_CONFIG),
-        'services': {
-            'First': copy.deepcopy(INITIAL_CONFIG),
-            'Second': copy.deepcopy(INITIAL_CONFIG),
-        },
-    },
-}
+CONFIG_KEYS = set(CHANGED_CONFIG.keys())
 
 
-CLUSTER_KEYS = list(INITIAL_CLUSTERS_CONFIG.keys())
+def get_config(adcm_object: AnyADCMObject):
+    """Get config or empty tuple (if config not defined)"""
+    try:
+        return adcm_object.config()
+    except ErrorMessage:
+        return ()
 
-SERVICE_NAMES = ['First', 'Second']
 
-
-@allure.step('Check cluster config')
-def assert_cluster_config(bundle: Bundle, statemap: dict):
-    for cname, clv in statemap.items():
-        actual_cnf = bundle.cluster(name=cname).config()
-        expected_cnf = clv['config']
-        for k, v in expected_cnf.items():
-            expect(
-                v == actual_cnf[k],
-                'Cluster {} config "{}" is "{}" while expected "{}"'.format(cname, k, str(actual_cnf[k]), str(v)),
-            )
-        for sname, service_expected_cnf in clv['services'].items():
-            service_actual_cnf = bundle.cluster(name=cname).service(name=sname).config()
-            for k, v in service_expected_cnf.items():
-                expect(
-                    v == service_actual_cnf[k],
-                    'Cluster {} service {} config {} is {} while expected {}'.format(
-                        cname, sname, k, str(service_actual_cnf[k]), str(v)
-                    ),
-                )
-    assert_expectations()
-
-
-@pytest.fixture()
-def cluster_bundle(sdk_client_fs: ADCMClient):
-    bundle = sdk_client_fs.upload_from_fs(get_data_dir(__file__, "cluster"))
-    for name in INITIAL_CLUSTERS_CONFIG:
-        cluster = bundle.cluster_create(name)
-        cluster.service_add(name='First')
-        cluster.service_add(name='Second')
-    return bundle
+check_config_changed = build_objects_checker(
+    field_name='Config',
+    changed=CHANGED_CONFIG,
+    extractor=get_config,
+)
 
 
 @pytest.fixture()
-def keys_clusters():
-    result = []
-    for i, key in enumerate(KEYS):
-        result.append((key, CLUSTER_KEYS[i % 3]))
-    return result
+def two_clusters(request, sdk_client_fs: ADCMClient) -> Tuple[Cluster, Cluster]:
```

Comment:
can't use create_two_clusters from utils here?

Reply:
Yes, forgot to change, thanks